applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -45,6 +47,11 @@ import { CodeBranchIcon } from "../components/icons";
 
 
 const useStyles = makeStyles((theme) => ({
+  infoIcon: {
+    fontSize: "0.9rem",
```

Comment:
Can use `fontSize="small"` attribute on material icons

Reply:
This is what I'd used initially, but it looked quite large. I'll revert to that, and then we can see what to do:

https://github.com/OpenSourceBrain/OSBv2/pull/351/commits/d45528d3e4aaa59e585e8bd00be1f433dd523b66#diff-00942284944a2e27c07669c41d8d514711a53531953f68def391bfd267e05247R418